### ISSUE TITLE
Boot Issue - Some pointers would be much appreciated

### ISSUE THREAD

**Issue by danriches** [michalin/ZX2022#2]
Hi,

I've had the pcb's made and I think the TMS9918's I bought are fake as it seems to kill the bus. I removed the video board and after connecting the Arduino Mega I loaded the rom.img into the board. I had several other issues so I then replaced the Z80 with a known working one then programmed the rom.img file again. Now it boots up and only shows a two line message and then seems to get stuck, the message is: 
"ZX2020 CP/M 3.1. Have fun."
"Looking for hard drive... found."

Now the found message changes to not found if I remove my CF to IDE adapter so that seems to work. However I never get a booted ready prompt no matter what I do. Do you have any pointers as to what I should do? Do I need to recompile CP/M as I don't have the video board connected anymore?

I'm desperate to get this running as it'd be so much fun and I can use it as a platform to teach my boys some low level coding.

Thanks Doctor Volt and thanks for a great project that I'm certain works fine, it's just I must be doing something wrong!!

Thanks, Dan Riches

**Comment by danriches** [comment]
I had two issues, the first was a dodgy SRAM chip and the second was that using the autorun function caused it to stop booting properly for some reason. Anyway it's all working now, I just need to find a compatible CF card as all mine either fail completely or fall over when using the pip c:=*.* command with "ERROR: CLOSE FILE  - C:COPYSYS.$$$". I'll try copying one file at a time and see if that works...

**Comment by michalin** [comment]
Did you format the CF card with the "mkdsk.com" utility? This writes the needed files to the CF card. You can watch the video from 7:03 for instructions: https://youtu.be/lTf1Y5tybfo?t=423

**Comment by danriches** [comment]
I did and that runs properly every time, I think I need to order another CF card and test it again. I did connect an old 60Gb PATA drive but that reports back as not supporting 8 bit mode which is a real pity. It seems the only ones that do support that mode are lower capacity ones and slightly older.

I think I might try to turn a raspberry pi Pico into a serial file server, it would be a fun project and I can update it via USB too. Although this would mean a more modern device attached to the retro style Z80 board and doesn't feel the best solution. I'll try both options I think.

Thanks for such a great little board design and your YouTube videos that go along with it, I've already subscribed to your channel and I can't wait to see what else you add or make. Keep up the great work!!

Thanks, Dan
